nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,111 @@
+This page contains general information about the client library to support you in the development of using The Microsoft Teams Client SDK.
 
-> [!TIP]
-> Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building both Teams-only and Microsoft 365-enabled apps and for current SDK capability support across Teams, Outlook, and Office.
+| Library | Description | APIs |
+|-|-|-|
+| **Live Share** | Live Share transforms the Microsoft 365 platforms to be more a collaborative experience without writing any dedicated back-end code | [Live Share](../docs-ref-autogen/%40microsoft/live-share/index.yml)<br/><br/>[Live Share canvas](../docs-ref-autogen/%40microsoft/live-share-canvas/index.yml)<br/><br/>[Live Share media](../docs-ref-autogen/%40microsoft/live-share-media/index.yml) |
+| **TeamsJS** | Teams Java script client library enables developers to create app content hosted in [IFrame](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) across the Microsoft 365 ecosystem | [TeamsJS ](../docs-ref-autogen/%40microsoft/teams-js/index.yml) |
+| **TeamsFx** | TeamsFx helps to reduce tasks by using Microsoft Teams single sign-on (SSO) and accessing cloud resources down to single line statements with little configuration | [Teamsfx](../docs-ref-autogen/%40microsoft/teamsfx/index.yml) |
 
-## Finding the library
+## Microsoft Teams Live Share
 
-The TeamsJS client is distributed as an npm package. The latest version can be found here:
+The [Live share SDK]( https://github.com/microsoft/live-share-sdk) is enabled to allow for collaborative moments across Microsoft Teams through `sidePanel` and `meetingStage`, media synchronization for HTML `<video>` and `<audio>` elements, and annotation tools for whiteboard and PowerPoint presentations. 
 
-www.npmjs.com/package/@microsoft/teams-js
+### Finding the library
 
-## Installing the library
+The Live share SDK is a JavaScript package comprised of three sub-packages ([Live Share](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md), [Live Share media](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-media-capabilities.md), and [Live Share canvas](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md)) which is published on npm and can be installed using npm or yarn.
 
-You can install the package using npm or yarn:
+#### Live share
 
-* `npm install --save @microsoft/teams-js`
-* `yarn add @microsoft/teams-js`
+To install the latest version of the Live share package to your application:
```

Comment:
Maybe consider summarizing and then Tab the installs to make more condensed or group based on npm vs yarn? Depends a bit on if you'd install these standalone or always all together.